unison-cli/unison/Main.hs

```diff
@@ -71,10 +75,10 @@ main :: IO ()
 main = withCP65001 do
   interruptHandler <- defaultInterruptHandler
   withInterruptHandler interruptHandler $ do
+    forkIO initHTTPClient
```

Comment:
From the docs it says it can be "an expensive operation", I forked it here just so it doesn't hold up the main thread at all or hurt loading times, it will still almost certainly complete before any HTTP calls are made.

All that said, I really have no reason to believe it's noticeably slow so I'm also completely fine running it inline.